Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -235,9 +239,8 @@ public void deleteIndexes() throws LuceneException {
 
             for (IWordID wordId : wordIdds) {
                 IWord word = dict.getWord(wordId);
-                List<ConceptEntry> entries = databaseClient.getConceptByWordnetId(word.getID().toString());
-                if (entries != null && !entries.isEmpty()) {
-                    logger.debug("Found concept for " + wordId.toString());
+                if (wordnetIdsOfWrappers.contains(word.getID().toString())) {
```

Comment:
This was changed because in the previous scenario each concept wrapper will be associated with only one wordnet concept. But from now on since the concepts are getting merged, there are chances three Wordnet concepts can be merged to a single concepts. So in that case we will have three wordnet ids stored in the wordnet variable separated by commas in ConceptEntry. Say now CCP-1 is linked to WID-1, WID-2 and WID-3. When we try to fetch the concept entry based on the wordnet id WID-1, we will not be able to retrieve CCP-1 since we search only by WID-1 but the wordnet id field will contains WID-1, WID-2, WID-3. So the design has been changed to fetch the wordnetIdsOfWrappers and storing it on a Set and by using contains we can make sure whether the wordnet id needs to be indexed or not. If the wordnet id is having a wrapper, there is no need to index it.